app/helpers/markdown_record/application_helper.rb

```diff
@@ -1,4 +1,27 @@
 module MarkdownRecord
   module ApplicationHelper
+    def method_missing(method, *args, &block)
```

Comment:
This change allows all helpers to be used in ERBs without chaining them onto `main_app`.